api-schema/src/main/resources/app.properties

```diff
@@ -0,0 +1 @@
+version=%APP_VERSION_TOKEN%
```

Comment:
💡 This is read by `Metadata.java` and used as the project metadata. Should we call this `metadata.properties` or `project_info.properties`?

This is not blocking. Feel free to merge after your decision.

Reply:
I like that a lot! I'll proceed with the change, thank you!